ttn_gw_data.json

```diff
@@ -0,0 +1,263 @@
+{
+	"1dee0b64b020eec4": {
+       	 	"alias": "Meetjestad #1 (De WAR)",
+		"latitude": 52.16270314,
+		"longitude": 5.37691284,
+		"altitude": 20
+	},
+	"mjs-gateway-1": {
+		"alias": "Meetjestad #1 (De WAR)",
+		"latitude": 52.16270314,
+		"longitude": 5.37691284,
+		"altitude": 20
+	},
+	"mjs-gateway-3": {
+		"alias": "Meetjestad #3 (Berghotel)",
+		"latitude": 52.14364015,
+		"longitude": 5.36431811,
+		"altitude": 40
+	},
+	"eui-1dee1cc11cba7539": {
+		"alias": "Meetjestad #4 (De Koperhorst)",
+		"latitude": 52.17026767,
+		"longitude": 5.39317479,
+		"altitude": 20
+	},
+	"mjs-gateway-4": {
+		"alias": "Meetjestad #4 (De Koperhorst)",
+		"latitude": 52.17026767,
+		"longitude": 5.39317479,
+		"altitude": 20
+	},
+	"1def5a0000000202": {
+		"alias": "Meetjestad #5 (Schothorst)",
+		"latitude": 52.17200,
+		"longitude": 5.39119,
+		"altitude": 5
+	},
+	"mjs-gateway-5": {
+		"alias": "Meetjestad #5 (Schothorst-W)",
+		"latitude": 52.17200,
+		"longitude": 5.39119,
+		"altitude": 5
+	},
```

Comment:
Deze twee zou ik eruit halen, dit is een tijdelijke gateway voor ontwikkeling die overal kan zijn.